### PR TITLE
Remove "System.out.println" from code

### PR DESCRIPTION
**Pull request by Liubomyr680**
1)Remove "System.out.println" from code
2)Fix DB_list checking

### CODE REVIEW

File: src/main/java/com/example/parser/utils/Parser.java [hunk 1]
```diff
@@ -14,13 +16,13 @@
...
 public class Parser {
 
-    // private static final String filePath = "C:\\Users\\Любомир\\Downloads\\dozvoli-na-vikidi-v-atmosferu-onovlenii.json";
     private PermitForEmissionsOfPollutants permit;
     private final PermitRepository permitRepository;
     private float time;
     private Date date;
     private int File_counter;
     private int DB_counter;
+    private static Logger log = LoggerFactory.getLogger(Parser.class);
```
Comment (vshynkar) [inline src/main/java/com/example/parser/utils/Parser.java]: Make it `private static final` as constant

File: src/main/java/com/example/parser/utils/Parser.java [hunk 2]
```diff
@@ -39,46 +41,34 @@ public void parser(String filePath) throws IOException, ParseException {
...
-//
-//        JsonList.add(permit);
-
         Set<PermitForEmissionsOfPollutants> union = new HashSet(JsonList);
         union.removeAll(BD_list);
 
-        for (PermitForEmissionsOfPollutants k : union)
+        for (PermitForEmissionsOfPollutants k : union) {
+            permitRepository.save(k);
             DB_counter++;
```
Comment (vshynkar) [inline src/main/java/com/example/parser/utils/Parser.java]: Variable name should start from the lower case and without underscore

File: src/main/java/com/example/parser/utils/Parser.java [hunk 3]
```diff
@@ -39,46 +41,34 @@ public void parser(String filePath) throws IOException, ParseException {
...
+        }
 
         long end = System.currentTimeMillis();
         time = TimeUnit.MILLISECONDS.toSeconds(end - start);
         date = new Date();
 
-        System.out.println("1) Затрачений час на обробку файлу " + time + " секунд" +
+        log.info("\n1) Затрачений час на обробку файлу " + time + " секунд" +
```
Comment (vshynkar) [inline src/main/java/com/example/parser/utils/Parser.java]: Use one log.info for one log line. So split it. Also, use placeholders here like 
`log.info("1. Затрачений час на обробку файлу {} секунд", time);`
And use ONLY English for all comments and messages.

**Review by vshynkar — CHANGES_REQUESTED** [review]

File: src/main/java/com/example/parser/utils/Parser.java (hunk 1 above)
Comment (Liubomyr680) [reply src/main/java/com/example/parser/utils/Parser.java]: done

File: src/main/java/com/example/parser/utils/Parser.java (hunk 2 above)
Comment (Liubomyr680) [reply src/main/java/com/example/parser/utils/Parser.java]: done

File: src/main/java/com/example/parser/utils/Parser.java (hunk 3 above)
Comment (Liubomyr680) [reply src/main/java/com/example/parser/utils/Parser.java]: done

**Review by vshynkar — APPROVED** [review]

**Pull request merged**